ToDoList-frontend/src/environments/environment.prod.ts

```diff
@@ -1,14 +1,14 @@
 export const environment = {
   production: false,
-  localhostUrl: 'https://localhost:7247/api/',
+  assigneeEndpoint: 'Assignee',
   
-  bucketEndpoint: 'https://localhost:7247/api/Bucket/',
-  bucketColorEndpoint: 'https://localhost:7247/api/BucketColor/',
-  bucketCategoryEndpoint: 'https://localhost:7247/api/BucketCategory/',
+  bucketEndpoint: 'Bucket',
+  bucketColorEndpoint: 'Dictionary/bucketColors/all',
+  bucketCategoryEndpoint: 'Dictionary/bucketCategories/all',
 
-  bucketTaskEndpoint: 'https://localhost:7247/api/BucketTask/',
-  bucketTaskStatesEndpoint: 'https://localhost:7247/api/BucketTaskState/all',
-  bucketTaskPrioritiesEndoint: 'https://localhost:7247​/api​/BucketTaskPriority​/all',
+  bucketTaskEndpoint: 'BucketTask/',
+  bucketTaskStatesEndpoint: 'Dictionary/bucketTaskStates/all',
```

Comment:
Suffix /all nie jest potrzebny. Jeśli chcemy pobrać wszystkie, to endpoint powinien byc `/diictionaries/bucket-task-states`. Jesli chcemy pobrac jeden, to wtedy `/dictionaries/bucket-task-states/{id}`